js/scripts.js

```diff
@@ -0,0 +1,7 @@
+var votingButtons = document.getElementsByClassName("button");
+
+for (var buttonNum = 0; buttonNum < votingButtons.length; buttonNum ++){
```

Comment:
`votingButtons.length` better move to new variable, because in your code on every loop change this value is recounting once again,

before `++` remove space

Reply:
I've added new variable - votingButtonsLen